src/kubetop/_twistmain.py

```diff
@@ -72,10 +72,20 @@ def __call__(self):
         ]
 
         t = Twist()
+
+        log_flag = u"--log-file"
+        log_file = u"~/.kubetop.log"
+        app_name = u"kubetop"
+        if isinstance(argv[0], bytes):
+            # sys.argv[0] is bytes on Python 2
+            log_flag = log_flag.encode("ascii")
+            log_file = log_file.encode("ascii")
+            app_name = app_name.encode("ascii")
```

Comment:
So on Python 2 these should be byte strings and on Python 3 they should be unicode strings?  Sounds like they are "native" string.  Too bad the string literal syntax for a native string is ambiguous (it could be a native string or it could be an unported byte string).

The explicit unicode string with encode here bothers me ... But maybe it's correct?  The comment here doesn't help me much.  Why does it matter if sys.argv[0] is bytes on Python 2?  Are you just detecting Python version here?  Or is there something more subtle going on?

Reply:
OK, I changed the conditional to:
```
if str is bytes:
```
and reworded the comment.